python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```
@@ -105,6 +105,11 @@ $defs:
         isis_authentication_key:
           type: str
           description: Type-7 encrypted password.
+        isis_authentication_cleartext_key:
+          type: str
+          description: |
+            Cleartext password. `isis_authentication_key` takes precedence over this key.
```

Move the precedence to the existing keu `isis_authentication_key` and write it like it was done for others please for consistency.

```suggestion
            Cleartext password.
```

---

Done.